Jenkinsfile

```diff
@@ -238,7 +238,7 @@ pipeline {
         parallelsAlwaysFailFast()
     }
     environment{
-        image = default_image_name()
+        image = "miopen-v1"
```

Comment:
Revert this change, please, and move "miopen-v1" to `default_image_name()`, line 19.

Reply:
done

Reply:
That's what I meant https://github.com/ROCmSoftwarePlatform/MIOpen/pull/676/commits/926800fe3acd53f7ca19c5e90351a0efe0d4ca7e

Reply:
Please update your local branch from repo